homeassistant/components/tuya/binary_sensor.py

```diff
@@ -28,8 +29,8 @@ class TuyaBinarySensorEntityDescription(BinarySensorEntityDescription):
     # DPCode, to use. If None, the key will be used as DPCode
     dpcode: DPCode | None = None
 
-    # Value to consider binary sensor to be "on"
-    on_value: bool | float | int | str = True
+    # Value or values to consider binary sensor to be "on"
+    on_value: bool | float | int | str | Sequence = True
```

Comment:
I would just make this a list or set.

```suggestion
    on_value: bool | float | int | str | set[bool | float | int | str] = True
```

Reply:
Excellent suggestions. Thanks.